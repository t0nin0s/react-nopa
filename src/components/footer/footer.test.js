import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Footer from './footer'

test('Footer component renders as expected', () => {
  const component = shallow(<Footer  />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
