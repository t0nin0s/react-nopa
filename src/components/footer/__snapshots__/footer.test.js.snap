// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders as expected 1`] = `
<div
  className="wrapper"
>
  <div
    className="footer"
  >
    <p>
      © Zopa Limited 2017 All rights reserved. 'Zopa' and the
              Zopa logo are trade marks of Zopa Limited. Zopa is a member
              of CIFAS – the UK's leading anti-fraud association, and we are
              registered with the Office of the Information Commissioner
              (No. Z879078).
    </p>
    <p>
      Zopa Limited is incorporated in England & Wales
              (registration number 05197592), with its registered
              office at 1st Floor, Cottons Centre, Tooley Street,
              London, SE1 2QG. Zopa Limited is authorised and regulated
              by the Financial Conduct Authority, and entered on the
              Financial Services Register under firm registration
              number 563134.
    </p>
  </div>
</div>
`;
